src/networkcloud/azext_networkcloud/aaz/latest/networkcloud/cluster/_create.py

```diff
@@ -0,0 +1,1026 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "networkcloud cluster create",
+    is_experimental=True,
+)
+class Create(AAZCommand):
+    """Create a new cluster or update the properties of the cluster if it exists.
+
+    :example: Create or update single rack cluster
+        az networkcloud cluster create --name "clusterName" -g "resourceGroupName" --extended-location name=""/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.ExtendedLocation/customLocations/clusterManagerExtendedLocationName" type="CustomLocation" --location "eastus" --analytics-workspace-id "/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/microsoft.operationalInsights/workspaces/logAnalyticsWorkspaceName" --cluster-location "Foo Street, 3rd Floor, row 9" --cluster-service-principal application-id="12345678-1234-1234-1234-123456789012" password="password" principal-id="00000008-0004-0004-0004-000000000012" tenant-id="80000000-4000-4000-4000-120000000000" --cluster-type "SingleRack" --cluster-version "3.0.0" --compute-deployment-threshold type="PercentSuccess" grouping="PerCluster" value=90  --network-fabric-id "/subscriptions/fca2e8ee-1179-48b8-9532-428ed0873a2e/resourceGroups/resourceGroupName/providers/Microsoft.Network/virtualNetworks/test-vnet/subnets/default" --aggregator-or-single-rack-definition "{\"networkRackId\":\"/subscriptions/subscriptionId/resourceGroups/rgName/providers/Microsoft.Network/virtualNetworks/cmName/subnets/ClusterManagerSubnet\",\"rackSkuId\":\"/subscriptions/subscriptionId/providers/Microsoft.NetworkCloud/rackSkus/VLab_Single_DellR750_8C2M_x70r3_9\",\"rackSerialNumber\":\"b99m99r1\",\"rackLocation\":\"b99m99r1\",\"availabilityZone\":\"1\",\"storageApplianceConfigurationData\":[{\"rackSlot\":1,\"adminCredentials\":{\"username\":\"admin\",\"password\":\"admin\"},\"storageApplianceName\":\"name\",\"serialNumber\":\"serial\"}],\"bareMetalMachineConfigurationData\":[{\"bmcCredentials\":{\"password\":\"bmcPassword\",\"username\":\"root\"},\"bmcMacAddress\":\"AA:BB:CC:DD:E7:08\",\"bootMacAddress\":\"AA:BB:CC:F8:71:2E\",\"machineName\":\"lab00r750wkr1\",\"rackSlot\":2,\"serialNumber\":\"5HS7PK3\"},{\"bmcCredentials\":{\"password\":\"bmcPassword\",\"username\":\"root\"},\"bmcMacAddress\":\"AA:BB:CC:FD:DC:76\",\"bootMacAddress\":\"AA:BB:CC:F8:50:CA\",\"machineName\":\"lab00r750wkr8\",\"rackSlot\":11,\"serialNumber\":\"9M56PK3\"}]}" --tags key1="myvalue1" key2="myvalue2"
```

Comment:
Please use fake subscription id in examples. BTW recommend using shorthand syntax for complicated value.
```suggestion
        az networkcloud cluster create --name "clusterName" -g "resourceGroupName" --extended-location name=""/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.ExtendedLocation/customLocations/clusterManagerExtendedLocationName" type="CustomLocation" --location "eastus" --analytics-workspace-id "/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/microsoft.operationalInsights/workspaces/logAnalyticsWorkspaceName" --cluster-location "Foo Street, 3rd Floor, row 9" --cluster-service-principal application-id="12345678-1234-1234-1234-123456789012" password="password" principal-id="00000008-0004-0004-0004-000000000012" tenant-id="80000000-4000-4000-4000-120000000000" --cluster-type "SingleRack" --cluster-version "3.0.0" --compute-deployment-threshold type="PercentSuccess" grouping="PerCluster" value=90  --network-fabric-id "/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.Network/virtualNetworks/test-vnet/subnets/default" --aggregator-or-single-rack-definition "{networkRackId:'/subscriptions/subscriptionId/resourceGroups/rgName/providers/Microsoft.Network/virtualNetworks/cmName/subnets/ClusterManagerSubnet',rackSkuId:'/subscriptions/subscriptionId/providers/Microsoft.NetworkCloud/rackSkus/VLab_Single_DellR750_8C2M_x70r3_9',rackSerialNumber:b99m99r1,rackLocation:b99m99r1,availabilityZone:1,storageApplianceConfigurationData:[{rackSlot:1,adminCredentials:{username:'admin',password:'admin'},storageApplianceName:name,serialNumber:serial}],bareMetalMachineConfigurationData:[{bmcCredentials:{password:'bmcPassword',username:'root'},bmcMacAddress:'AA:BB:CC:DD:E7:08',bootMacAddress:'AA:BB:CC:F8:71:2E',machineName:lab00r750wkr1,rackSlot:2,serialNumber:5HS7PK3},{bmcCredentials:{password:'bmcPassword',username:'root'},bmcMacAddress:'AA:BB:CC:FD:DC:76',bootMacAddress:'AA:BB:CC:F8:50:CA',machineName:lab00r750wkr8,rackSlot:11,serialNumber:9M56PK3}]}" --tags key1="myvalue1" key2="myvalue2"
```

Reply:
Updated to fake subscriptionId and shorthand syntax.